package.json

```diff
@@ -61,6 +61,9 @@
           "module.inventory.enabled",
           "inventory.instances.item.post",
           "inventory-storage.instances.item.post",
+          "inventory.items.item.post",
+          "inventory-storage.holdings.item.post",
+          "ui-inventory.holdings.create",
```

Comment:
Pretty sure we don't want `ui-inventory.holdings.create`; it makes this plugin's functionality directly dependent on ui-inventory being installed. It looks like [`ui-inventory.holdings.create`](https://github.com/folio-org/ui-inventory/blob/71e18ba3f3083fe0ebd81e7149678ea4f8a39d8b/package.json#L534-L542) just contains view-related things plus `inventory-storage.holdings.item.post`, which you have already included above. 

If you're confident we need it, I think the right approach is to sift through the permissions in `ui-inventory.instance.view` to figure out which individual entries from that pset are omitted in this one, and then add them here.

Reply:
I removed that permission and tested again, and you're right, we don't need it.  Since I was messing around with permissions anyway, I went ahead and removed the permission you thought might be erroneously granting access to inventory, and you were right about that too.  Removing inventory enable removes the inventory button when you only have fast add permissions.